src/main/java/com/spartronics4915/frc2019/subsystems/LED.java

```diff
@@ -17,25 +22,35 @@ public static LED getInstance()
         return mInstance;
     }
 
-    public enum WantedState
+    // private enum SubsytemLEDState
+    // {
+    //     CLIMBING, 
+    // }
+
+    private final byte[] kForwards = "2".getBytes();
+    private final byte[] kBackwards = "3".getBytes();
+    private final byte[] kOff = "4".getBytes();
+    private final byte[] kAutonomous = "5".getBytes();
+
+    private enum LEDState
```

Comment:
Why isn't this needed?

Reply:
First of all, even if you do use this, there's a bug where the LED strip will work on the first start, but won't work after this subsystem's `stop` method is called.

Second, we don't need `LEDState` because:
1. We want to have LEDs on at all times the robot is on.
2. It's redundant. `DriveLEDState` already has an `OFF` state.
3. Even if we add something other than `DriveLEDState` (e.g. `DecorativeLEDState`), we can just set both to `OFF` manually.